packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/materials/pbrMaterialPropertyGridComponent.tsx

```diff
@@ -28,6 +28,9 @@ interface IPBRMaterialPropertyGridComponentProps {
     onPropertyChangedObservable?: Observable<PropertyChangedEvent>;
 }
 
+/**
+ * @internal

```

Comment:
@RaananW I added this because the linter was complaining. Is this ok?

Reply:
It's not really internal but it won't make any major difference here. I guess you could have left it as is (linter just shows you a warning), or document is by explaining what the component is doing. But this should probably not be a part of this PR